### DISCUSSION TITLE
Is it depreciated ? if yes what is alternative

### DISCUSSION THREAD

**Discussion by herawais** [adrien2p/medusa-extender#197] [Q&A]
Is it depreciated for latest medusa version (1.12) ? if yes what is reason and alternative ?

**Comment by SGFGOV** [comment]
Hi you can try @sgftech/medusa-extender interim

On Fri, 9 Jun, 2023, 16:37 Muhammad Awais, ***@***.***> wrote:

> Is it depreciated for latest medusa version (1.12) ? if yes what is reason
> and alternative ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adrien2p/medusa-extender/discussions/197>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXEQJHG5LYBU44MF7VL4JALXKL7V5ANCNFSM6AAAAAAZAP4T2Y>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 

This message (including any attachments) may contain confidential, 
proprietary, privileged and/or private
information. The information is 
intended to be for the use of the individual or entity designated above. If
you are not the intended recipient of this message, please notify the 
sender immediately, and delete the
message and any attachments. Any 
disclosure, reproduction, distribution or other use of this message or
any 
attachments by an individual or entity other than the intended recipient is 
prohibited.

**Reply by herawais** [reply]
@SGFGOV thanks for sharing, do you have any idea how to actually get working the multi tenancy in it ?

**Comment by SGFGOV** [comment]
I am not sure about multi tenancy..but if you are looking to create a
marketplace you can follow the old tutorial .. with easy replacements as
shown in the start demo which is in the repo

On Sat, 10 Jun, 2023, 12:21 Muhammad Awais, ***@***.***>
wrote:

> @SGFGOV <https://github.com/SGFGOV> thanks for sharing, do you have any
> idea how to actually get working the multi tenancy in it ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adrien2p/medusa-extender/discussions/197#discussioncomment-6138647>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXEQJHDG2YN4IOES5OODTKLXKQKQPANCNFSM6AAAAAAZAP4T2Y>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>

-- 

This message (including any attachments) may contain confidential, 
proprietary, privileged and/or private
information. The information is 
intended to be for the use of the individual or entity designated above. If
you are not the intended recipient of this message, please notify the 
sender immediately, and delete the
message and any attachments. Any 
disclosure, reproduction, distribution or other use of this message or
any 
attachments by an individual or entity other than the intended recipient is 
prohibited.
